docs/API.md

```diff
@@ -26,6 +26,8 @@
 * state:onInput( function(self,fields,playername)) - specify a function to run after data and/or events received
 * state:button( x,y,w,h,name,text [, exit_on_click] ) - create a new button at x,y with name and caption (text)
  * ^ optional: exit_on_click - set to true to exit the form when the button is clicked. ( Also see button.setClose() )
+   state:image_button( x,y,w,h,name,text, image [, exit_on_click] ) create a new button with image.
```

Comment:
Missing asterisk (`*`)? Same below.

As-is:

* state:button( x,y,w,h,name,text [, exit_on_click] ) - create a new button at x,y with name and caption (text)
￼   * ^ optional: exit_on_click - set to true to exit the form when the button is clicked. ( Also see button.setClose() )
￼   state:image_button( x,y,w,h,name,text, image [, exit_on_click] ) create a new button with image.
   state:item_image_button( x,y,w,h,name,text, item [, exit_on_click] ) create a new button with item as image.

With asterisks:

* state:button( x,y,w,h,name,text [, exit_on_click] ) - create a new button at x,y with name and caption (text)
￼   * ^ optional: exit_on_click - set to true to exit the form when the button is clicked. ( Also see button.setClose() )
* state:image_button( x,y,w,h,name,text, image [, exit_on_click] ) create a new button with image.
* state:item_image_button( x,y,w,h,name,text, item [, exit_on_click] ) create a new button with item as image.